vmdb/config/permissions.yml

```diff
@@ -0,0 +1,4 @@
+---
+- permission one
+- permission two
+- miq_report
```

Comment:
We might need a way to map high level features/areas to the individual permissions. This gives us a good start for discussion. I like the use of a null store for testing.